src/test/java/io/strimzi/kafka/bridge/clients/BasicKafkaClient.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.clients;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.strimzi.kafka.bridge.utils.KafkaJsonSerializer;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.time.Duration;
+import java.util.Properties;
+import java.util.Random;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.IntPredicate;
+
+public class BasicKafkaClient {
+
+    private final String bootstrapServer;
+
+    public BasicKafkaClient(String bootstrapServer) {
+        this.bootstrapServer = bootstrapServer;
+    }
+
+    /**
+     * Send messages to external entrypoint of the cluster with PLAINTEXT security protocol setting
+     * @return sent message count
+     */
+    public int sendMessagesPlain(long timeoutMs, String topicName, int messageCount, String message, int partition,
```

Comment:
this client will send messages to the Kafka cluster where the bridge is connect as well. You are using "plain" just because maybe this code comes from the strimzi operator or because you want to add a TLS sender to the Kafka cluster? Do you see any value in this?

Reply:
Of course. In the future, we will be able to use TLS as well as tracing and OAuth clients.